data_steward/validation/participants/identity_match.py

```diff
@@ -741,6 +785,23 @@ def write_results_to_drc_bucket(project, validation_dataset=None):
                     )
 
 
+def _add_matches_to_results(results, matches, field):
```

Comment:
There is one more case that can occur here: The individual does not exist in `matches` but exists in `results`. The value for the field should be assigned to `consts.MISSING` for such individuals.

Reply:
You are right.  This case is handled by this function.  For example, when this gets triggered the first time, every individual's entry in the matches dictionary will run through the initialization condition for an individual.  This initialization condition sets every value in the individual's dictionary to the `consts.MISSING` value.  So, on the first name comparison, everything is set to `consts.MISSING` then the first name match values are assigned.  Now if person person 1 has first name data but no last name data, they will not be part of the matches dictionary sent to this function with the last name data.  This is fine because when the person dictionary definition was created all their fields were initialized to `consts.MISSING`.

Reply:
Yes, that makes sense. Thanks for clarifying that, Lauren.
I now have a question which need not be addressed in this pull request. If an individual does not exist in any of the `pii` tables, but does exist in the `participant_match` table, should they be included in the `identity_match` table with all fields marked as missing? Right now, it looks like they do not exist in the `identity_match` table, which I think is also fine. It just needs to be communicated to the sites and anyone querying the two match tables.